imports/plugins/core/system-info/server/no-meteor/schemas/schema.graphql

```diff
@@ -0,0 +1,16 @@
+extend type Query {
+  systemInformation: SystemInfo
```

Comment:
Add a string literal description of this query above it. Also add `!` after `SystemInfo` because there will always be a non-null return if there isn't an error thrown.

And as I mentioned in another comment, you'll need to add `(shopId: ID!)` param for this query.